AppCenterAnalytics/AppCenterAnalytics/Internals/Model/Properties/MSStringTypedProperty.m

```diff
@@ -0,0 +1,55 @@
+#import "MSStringTypedProperty.h"
```

Comment:
Please reorder. I know you can get confused but we haven't had a rule to specifying class header file at the top, there was a confusion. Just sort it alphabetically.

Reply:
I think we need to change that then, because with `#import`s, that can actually break certain files. @ElektrojungeMS

Reply:
(the order matters for imports in objective c, it's not like java or C#)